src/components/collapsible-section/collapsible-section.scss

```diff
@@ -73,13 +73,24 @@
 
 .section__header__title {
     @include lime-typography.typography(headline2);
-    @include mixins.truncate-text;
-    line-height: normal;
 
     justify-self: flex-start;
     padding-right: functions.pxToRem(12);
 
     user-select: none; // mostly to improve experience on Android, where tapping on sections selects the text too
+
+    padding-right: functions.pxToRem(8);
+
+    // Below tries to render text in two lines,
+    // and then truncate if there is no more space
+    height: auto;
+    max-height: 3rem;
```

Comment:
yes kind of… But `-webkit-line-clamp` is supported by very old browsers I think. So it's not a big deal for me if we increase it to fit in 3 lines in those case, or set it on 2.4rem.

Reply:
I was more thinking of non-webkit browsers… but I guess, at least the CRM Web Client, strictly speaking, doesn't actually support any non-webkit browsers 😄

Let's keep it like this until we get any requests to change it (if we ever do). It's functional. I think it's better than cutting the text cleanly if there is no ellipsis, since there really needs to be _some_ indication that the text has been truncated.